torchvision/transforms/functional.py

```diff
@@ -714,7 +714,7 @@ def adjust_saturation(img: Tensor, saturation_factor: float) -> Tensor:
     return F_t.adjust_saturation(img, saturation_factor)
 
 
-def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:
```

Comment:
Technically speaking, the type of `hue_factor` is `torch.Tensor` of `dtype=whatever_torch.rand()'s_output_type_is`, but I guess it would be misleading to put `hue_factor: Tensor` in the header

Reply:
Also, since we're in `functional.py`, the type of `img` would be something like `Union[Tensor, PIL]`, but I cannot use `PIL` as a type so I left `img` as it is. Not sure what to do. I don't think putting `Any` is correct either.

Reply:
Here the type hints are mostly for `torch.jit.script` which does not accept type mixing with `Union`. Let's keep annotations as it is done for other. This is correct`def adjust_hue(img: Tensor, hue_factor: float) -> Tensor:`